transports/msmq/index.md

```diff
@@ -156,6 +156,10 @@ See also [Message Queuing Security Overview](https://docs.microsoft.com/en-us/pr
 
 ## Distributed Transaction Coordinator
 
-To support guaranteed [once delivery of messages,](/nservicebus/operations/transactions-message-processing.md) NServiceBus makes use of the Distributed Transaction Coordinator (DTC) to synchronize transactions between MSMQ and the database. For this to work, the DTC must be started and configured correctly.
+In order to support [guaranteed once delivery of messages],(/nservicebus/operations/transactions-message-processing.md), NServiceBus makes use of the Distributed Transaction Coordinator (DTC) to synchronize transactions between MSMQ and the database. For this to work, the DTC must be started and configured correctly. This can be done manually or with the [NServiceBus PowerShell module](/nservicebus/operations/management-using-powershell.md).
```

Comment:
```suggestion
In order to support [guaranteed once delivery of messages](/nservicebus/operations/transactions-message-processing.md), NServiceBus makes use of the Distributed Transaction Coordinator (DTC) to synchronize transactions between MSMQ and the database. For this to work, the DTC must be started and configured correctly. This can be done manually or with the [NServiceBus PowerShell module](/nservicebus/operations/management-using-powershell.md).
```

Reply:
This first sentence feels a little awkward. Would it make sense to swap it around?

> NServiceBus makes use of the Distributed Transaction Cordinator (DTC) to synchronize transactions between MSMQ and the database in order to support [guaranteed once deliver of messages](#).

Reply:
Also, in the section below we talk about MSDTC. Should we use that same term and abbreviation here?